### ISSUE TITLE
JAWS in auto forms mode announces the link href of a menu button

### ISSUE THREAD

**Issue by mitchellevan** [FreedomScientific/VFO-standards-support#56]
### Summary

1. Use JAWS with auto forms mode enabled. (Note: the issue is not reproducible when auto forms mode is turned off.)
2. Use IE11.
3. Go to https://labs.levelaccess.com/images/2/25/Link-as-aria-menu-button.html
4. Tab to the "Log Out" menu button.

Note: the "Log Out" menu button is coded like this:
`<a role="button" aria-expanded="false" aria-haspopup="true" href="javascript:void(0)">Log Out</a>`

### Expected result

This element should be named by its content ("Log Out"), and the JAWS should not announce the href value of the link.

### Actual result

JAWS announces both the element content ("Log Out") and the href value.

Pasted from JAWS Speech History:

Log Out button menu 
javascript:void(0)
Press space to activate the menu, then navigate with arrow keys

### Example

See steps above for a code example.

#### JAWS version and build number

JAWS 2018.1802.78
 
#### Operating System and version

OS Name	Microsoft Windows 10 Pro
Version	10.0.16299 Build 16299

#### Browser and version:

Internet Explorer
11.248.16299.0

**Comment by stevefaulkner** [comment]
@mitchellevan is there a particular reason why the JavaScript is in the HREF?

**Comment by mitchellevan** [comment]
Without ARIA, the A tag would need an HREF in order to have a link role. When the link has only JavaScript behavior the javascript:void(0) is one of the most common "do nothing" values for the HREF. I suppose in this context it's a non-ARIA fallback.

**Comment by scottaohara** [comment]
this still happens in JAWS 2020, but this seems a low priority issue that could be easily mitigated by an author slightly adjusting their code.

the button element could be used here instead of a link with role=button.

the href could be removed and a tabindex=0 could be used instead to allow this element to receive keyboard focus.  

regarding a non-ARIA fallback would insinuate that the ARIA is applied via JavaScript, in which case then the href also makes no sense.   Since ARIA is supported in IE11 though, there's no reason as to need to support a non-ARIA fallback otherwise.

Closing and archiving this issue as there are work arounds an author could do to mitigate this announcement, and those work arounds would largely involve little effort while also resulting in better markup.
